### PR TITLE
FIX

### PR DESCRIPTION
**Pull request by ducit95**

### CODE REVIEW

File: tests/UserTest.php [hunk 1]
```diff
@@ -12,39 +12,72 @@ class UserTest extends TestCase
...
     {
-        $response = $this->call('GET', 'detele/10');
-
-    	$this->assertEquals(200, $response->status());
+        $response = $this->call('POST', 'create', [
+          'name' => 'Duc Nguyen',
+          'address' => 'Ha Noi',
+          'age' => '50'
+        ]);
+        $this->assertRedirectedToAction('AppController@store');
```
Comment (anh-pham191) [inline tests/UserTest.php]: This assertion only use for asserting into this store method but we can not see if data have been imported to the database yet or not. Can you try another assertion method to check the database if this valid user exist or not ?

File: tests/UserTest.php [hunk 2]
```diff
@@ -12,39 +12,72 @@ class UserTest extends TestCase
...
     {
-        $response = $this->call('POST', 'create',['name' => 'Taylor','address' =>'HA NOI','age'=>6]);
-
-    	$this->assertEquals(200, $response->status());
+        $response = $this->call('POST', '/create', [
+          'name' => 'Duc Nguyen',
+          'address' => 'hanoi',
+          'age' => '10'
+        ]);
+        $this->assertFalse(false);
```
Comment (anh-pham191) [inline tests/UserTest.php:102]: Not enough test case I suppose, can you check the requirement of test app, in here i can only find the empty case

File: tests/UserTest.php (hunk 1 above)
Comment (ducit95) [inline tests/UserTest.php]: in AppController i check if isset $id ->delete($id)

File: tests/UserTest.php (hunk 2 above)
Comment (ducit95) [inline tests/UserTest.php]: all validate i use angularjs so not the empty case

Comment (ducit95) [inline tests/UserTest.php]: i did fix it

**Comment by ducit95** [comment]
i did fix it you can check Update UserTest.php

**Pull request closed without merging**